1-js/08-prototypes/02-function-prototype/1-changing-prototype/task.md

```diff
@@ -54,7 +53,7 @@ alert( rabbit.eats ); // true
     alert( rabbit.eats ); // ?
     ```
 
-3. And like this (replaced one line)?
+3. E se for assim (a linha foi substituída)?
```

Comment:
> Frases seguidas começadas com a mesma palavra [PORTUGUESE_WORD_REPEAT_BEGINNING_RULE](https://community.languagetool.org/rule/show/PORTUGUESE_WORD_REPEAT_BEGINNING_RULE?lang=pt)

- Two sentences in a row begin with the same adverb. 
- Consider rephrasing the phrase or use a dictionary to find a synonym.
- **Suggestion**: change `E` to `Ou`

<img width="677" alt="image" src="https://user-images.githubusercontent.com/35238611/177468374-5f94614b-23f3-43ef-b3f2-08459fa246ad.png">